ynput/OpenPype
# Houdini: OpenGL Review support LOPs/Solaris source mode

## Changelog Description

Allow reviews using the openGL node to use LOP Paths.

## Additional info

Based on comment [here](https://github.com/ynput/OpenPype/issues/4869#issuecomment-1517572996) that OpenGL review node publishing didn't support Solaris reviews yet using LOP paths.

TODO:

- [ ] Update creator to allow 'use selection' for LOP paths and then have it auto-set to LOPs source mode.
- [x] Update collector to collect focal lengths for LOPs camera
  - [ ] Make sure it always collects the correct data (see todo in code)
- [ ] Update validator to allow LOPs scene path, etc.

## Testing notes:

1. Set up openGL review instance to use LOP path
2. Publish review
3. Set up openGL review instance to use OBJ camera path
4. Publish review

Make sure to publish with reviews enabled so it also tests the focal length burnin.
